### PR TITLE
security doc

### PR DESCRIPTION
**Pull request by mattmanning**
This PR adds a new doc that explains the KernelUser permissions lockdown effort as well as some other security benefits of using Convox.

## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by mattmanning** [comment]
[Security doc](https://trello.com/c/HVFNSXvx/1593-security-doc)

File: _reference/security.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Security"
+---
+
+Convox always has your security in mind. Below are some of the ways your apps and resources are protected.
+
+## AWS Isolation
+
+Your Convox Rack is installed in your own AWS account. Unlike multi-tenant Paas platforms, you never have to worry about other users gaining access to your infrastructure.
```
Comment (nzoschke) [inline _reference/security.md]: PaaS

File: _reference/security.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- Access to DynamoDB tables that only begin with the name of the Rack as a prefix
+- Access to RDS instances that only begin with the name of the Rack as a prefix
+
+## VPC Isolation
+
+All of the infrastructure that Convox creates for you runs inside a Virtual Priate Cloud ([VPC](https://aws.amazon.com/vpc/)). This provides additional isolation at the networking layer. By default, all resources such as datastores are created in such a way that they can only be access from inside the VPC.
+
+## Private Networking
+
+If you'd like to take network isolation one step further you can run your Rack in private networking mode, where the Rack instances run in private subnets that access the Internet through NATs. Read more on the private networking [doc](/docs/private-networking/).
```
Comment (nzoschke) [inline _reference/security.md]: style alert: Don't capitalize internet any more

http://www.poynter.org/2016/ap-style-change-alert-dont-capitalize-internet-and-web-any-more/404664/

Comment (nzoschke) [inline _reference/security.md]: Should also mention that the internet can not access rack instances

File: _reference/security.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+title: "Security"
+---
+
+Convox always has your security in mind. Below are some of the ways your apps and resources are protected.
+
+## AWS Isolation
+
+Your Convox Rack is installed in your own AWS account. Unlike multi-tenant Paas platforms, you never have to worry about other users gaining access to your infrastructure.
+
+[Convox Console](https://console.convox.com) is a multi-tenant management layer that proxies API requests to your Rack and provides team management and workflow tools. If you don't want to use the multi-tenant Console you can run your own! [Contact us](mailto:support@convox.com) for information about plans and pricing.
```
Comment (nzoschke) [inline _reference/security.md]: maybe something about API keys, user roles and deploy keys?

File: _reference/security.md (hunk 1 above)
Comment (nzoschke) [inline _reference/security.md]: and "PaaS platform" is redundant.

File: _reference/security.md (hunk 2 above)
Comment (nzoschke) [inline _reference/security.md]: Some other topics to consider...

Dedicated Instances. Guarantees that there is no multi-tenancy on the AWS instance level.

Load Balancers. Reduces ingress to HTTPS traffic to a specific instance port and application service.

Comment (mattmanning) [inline _reference/security.md]: I'm torn on this capitalization issue. Oxford disagrees with AP http://blog.oxforddictionaries.com/2016/04/should-you-capitalize-internet/

It's a proper noun, so I feel like it should be capitalized, but I may be on the losing side of that battle.

**Comment by mattmanning** [comment]
@nzoschke Thanks for the great feedback. I'm still thinking about whether to decapitalize "Internet". I will probably do that in a separate change across the whole site.

I implemented all of your other suggestions.

**Pull request merged**